docs/products/clickhouse/concepts/databases-and-tables.rst

```diff
@@ -1,5 +1,5 @@
-Work with databases and tables
-===============================
+About databases and tables
```

Comment:
I usually use format How To + verb. Here it is How to About databases. Do we care about How To + verb? or not that much?

Reply:
I moved it to concepts, because it wasn't very task-based. So then the titles often are "About ...."